app-admin/pwman3/pwman3-0.9.9.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A lightweight password manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+PYTHON_COMPAT=( python3_5 python3_6 )
+inherit distutils-r1
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+sqlite mysql postgresql mongodb test"
```

Comment:
Order by flag name, please.

You aren't using IUSE=test.

Reply:
I wasn't sure about this ... The application's test suite is quite extensive, but it requires services like mongodb and postgresql running.
After reading about [testing in gentoo](https://devmanual.gentoo.org/ebuild-writing/functions/src_test/index.html) it is clear to me that my test suite needs a lot of changes to run in gentoo.
So, we'll have to count on travis CI, and assume that releases work fine on gentoo. 
In the future I will separate unittest tests from the integration tests such that gentoo users can run the tests without network based services.

Reply:
Would any useful tests remain if you disabled those relying on external services? If they have extensive suite, I suppose they also have tests using sqlite.